.github/workflows/mac-dmg.yml

```diff
@@ -15,8 +15,8 @@ on:
         type: boolean
 
 env:
-  JAVA_DIST: 'temurin'
-  JAVA_VERSION: 20
+  JAVA_DIST: 'zulu'
```

Comment:
needs to be tested due to https://github.com/cryptomator/cryptomator/issues/3030

Reply:
Tested [this build](https://github.com/cryptomator/cryptomator/actions/runs/6428885695), GUI loads just fine.